app/internal/auth/oauth/oauth.go

```diff
@@ -18,6 +18,7 @@ import (
 	"time"
 
 	"github.com/golangci/golangci-api/app/internal/auth/sess"
+	"github.com/golangci/golangci-worker/app/analytics"
```

Comment:
could not import github.com/golangci/golangci-worker/app/analytics (cannot find package "github.com/golangci/golangci-worker/app/analytics" in any of:
	/usr/local/go/src/github.com/golangci/golangci-worker/app/analytics (from $GOROOT)
	/tmp/tmp.UVbATgtbGw/src/github.com/golangci/golangci-worker/app/analytics (from $GOPATH))